### ISSUE TITLE
Impossible to specify read model storage adapter

### ISSUE THREAD

**Issue by timbset** [reimagined/resolve#582]
There is the mysql adapter for read model storage. But I have no way to use it.
Expected config part:
```json
"readStorage": {
  "adapter": "mysql",
  "options": {
    ...
  }
}
```

**Comment by MrCheater** [comment]
Maybe?
```json
{
	"eventStorage": {
		"adapter": "custom-event-storage-name",
		 "options": {
		 	 
		 }
	},
	"readModelStorage": {
		"adapter": "custom-read-model-storage-name",
		 "options": {
		 	 
		 }
	},
	"viewModelSnapshotStorage": {
		"adapter": "custom-view-model-snapshot-storage-name",
		 "options": {
		 	 
		 }
	}
}
```

**Comment by timbset** [comment]
I agree with you. Something similar is written in #583

**Comment by TatyanaRyzh** [comment]
Hi timbset,
 
We've reproduced your scenario with those config settings and will fix it as soon as possible.
 
Thank you for helping us improve reSolve!

**Comment by IhostVlad** [comment]
Implemented in [V0.12.3](https://github.com/reimagined/resolve/releases/tag/V0.12.3)
